notebooks/community/managed_notebooks/Fraud detection/Fraud Transaction Detection notebook.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>Data must live in gs://cloud-samples-data/vertex-ai/managed_notebooks/, please rename fraud_detection/PS_20174392719_1491204439457_log.csv to something that makes more sense and ask ivanmkc to move it there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>please find that updated file path is "gs://cloud-samples-data/vertex-ai/managed_notebooks/fraud_detection/fraud_detection_data.csv"</p>

Reply:
<p>gs://vertex_ai_managed_services_demo/fraud_detection/PS_20174392719_1491204439457_log.csv</p>